test.js

```diff
@@ -2,4 +2,9 @@
 
 // wrong codes
 var a = "1";
-a.includes("1");
+
+a.includes('1')
+
+for (var i = 0; i< 100; i++) { 
+ console.log('a' + i)
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ipf-jypark/test-repo/pullRequest?prid=7372446)
```suggestion
 console.log("a" + i)
```